templates/institutional-landing/institutional-landing-fr.html

```diff
@@ -0,0 +1,120 @@
+		<h2 class="wb-inv">Pleins feux sur</h2>
+		<p><a class="stretched-link" href="#">[Titre de la promotion]</a></p>
+	</div>
+</section>
+
+{%- include /components/gc-most-requested/gc-most-requested.html -%}
+
+<div class="container mrgn-bttm-lg">
+	<section class="gc-srvinfo">
+		<h2 class="wb-inv">Services et renseignements</h2>
+		<div class="row wb-eqht-grd">
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+			<div class="col-lg-4 col-md-6">
+				<h3><a href="#">[Lien hypertexte]</a></h3>
+				<p>Utilisez des verbes d'action ou répertoriez simplement des mots-clés pour résumer les informations ou les tâches qui peuvent être accomplies sur la page à laquelle elles renvoient.</p>
+			</div>
+		</div>
+	</section>
```

Comment:
Note: to add a new sample in gc-srvinfo for this 3 column configuration

https://github.com/wet-boew/GCWeb/tree/master/components/gc-servinfo/samples